### PR TITLE
Creating PersonName in ACSP data model

### PR DESCRIPTION
**Pull request by sampankumar**
__Short description outlining key changes/additions__
Creating PersonName in ACSP data model
__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/acsp/models/filing/PersonName.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PersonName {
+
+    @JsonProperty("first_name")
+    private String firstName;
+    @JsonProperty("last_name")
+    private String lastName;
+    @JsonProperty("middle_name")
+    private String middleName;
```
Comment (swalke-ch) [inline src/main/java/uk/gov/companieshouse/acsp/models/filing/PersonName.java:12]: These fields need to match with existing model in chips-filing-consumer
https://github.com/companieshouse/chips-filing-consumer/blob/master/src/main/java/uk/gov/ch/services/dataholdingobjects/source/filingtype/PersonName.java

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

**Pull request merged**